code/src/WijmoProvider/Features/Styling.ts

```diff
@@ -111,5 +111,21 @@ namespace WijmoProvider.Feature {
                 );
             }
         }
+
+        public setColumnWordWrap(columnID: string, value: boolean): void {
+            // validate if column exists
+            const column = this._grid.getColumn(columnID);
+            if (column) {
```

Comment:
I believe you can just do column.provider.worWrap = value;

Reply:
I'll do that